docker/Dockerfile.linux.amd64

```diff
@@ -1,16 +1,18 @@
-FROM alpine:3.6 as alpine
-RUN apk add -U --no-cache ca-certificates
+FROM alpine:3.9
+
+LABEL maintainer="Drone.IO Community <drone-dev@googlegroups.com>" \
+  org.label-schema.name="Drone Autoscaler" \
+  org.label-schema.vendor="Drone.IO Community" \
+  org.label-schema.schema-version="1.0"
 
-FROM alpine:3.6
```

Comment:
@tboerger, any reason to not use `FROM scratch` here and just import the certs from the alpine image?